### ISSUE TITLE
Switching from mirrored to unmirrored with different resolutions not working as expected.

### ISSUE THREAD

**Issue by philipsdean** [jakehilborn/displayplacer#20]
Moving from a mirrored / optimized resolution for external display to a unmirrored setup with separate resolutions caused my external display to default to my macbook's custom display resolution.

**Comment by jakehilborn** [comment]
I've noticed this happening sometimes too though I'm not able to reproduce it consistently.

**Comment by jakehilborn** [comment]
@philipsdean Mirroring behavior has been substantially improved in [release v.1.30](https://github.com/jakehilborn/displayplacer/releases). Displayplacer now explicitly un-mirrors before trying to change resolutions. Could you try out the new release and let me know if it works for you?

**Comment by philipsdean** [comment]
I tried it out on the new release and it is working as expected now!

Although it has been a few years so i'm not on the same OS version but I did try it on the same machine. I also went back and tried the old version again and it also now worked as expected. I actually found and used the same commands so perhaps something also has changed in macOS since Mojave or in my external displays firmware.
